lib/mutant/meta/example/dsl.rb

```diff
@@ -19,9 +19,10 @@ def self.call(file, block)
         # Initialize object
         #
         # @return [undefined]
-        def initialize(file)
-          @file     = file
-          @source   = nil
+        def initialize(file, type)
+          @file, @type = file, type
```

Comment:
For consistency I would probably just have each assignment on their own line, rather than using two different ivar assignment approaches in one method.